file-store/src/lib.rs

```diff
@@ -94,6 +94,14 @@ impl FileStore {
         Ok(())
     }
 
+    pub fn update_total_size(&self) -> Result<u64> {
+        let mut total = 0;
+        for uuid in self.all_uuids()? {
+            total += self.get_size(uuid?)?;
```

Comment:
I don't know enough context to answer this question by myself:

is there any risk that an update file gets deleted while we are counting the size, in normal operation? If so, is failing to get the size a sufficient reason to fail the entire sum, or should we rather unwrap_or_zero it?

Reply:
Yep good catch, it doesn’t look like a good reason to crash the whole stat computation.